go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/aws/karpenter-provider-aws
 
-go 1.23.5
+go 1.24rc2
```

Comment:
Marking that we need to revert this (though I expect that you will catch this)

Reply:
Yep, this will bump to 1.24.0 once I rebase.